hr_attendance_calendar/views/hr_attendance_day_view.xml

```diff
@@ -42,15 +42,19 @@
                                 <tree>
                                     <field name="check_in"/>
                                     <field name="check_out"/>
-                                    <field name="worked_hours" widget="float_time"/>
+                                    <field name="worked_hours"
```

Comment:
paid_hours?

Reply:
Nope. This refers to the worked_hours field from the attendance class.